pom.xml

```diff
@@ -256,6 +256,11 @@
       <artifactId>wordnet-random-name</artifactId>
       <version>1.5</version>
     </dependency>
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>okhttp</artifactId>
+      <version>5.0.0-alpha.11</version>
```

Comment:
is there a reason you aren't using the built-in Java HttpClient? 
https://www.baeldung.com/java-9-http-client

this avoids adding another dependency, either way probably shouldn't be using an alpha one